Dockerfile

```diff
@@ -35,7 +35,7 @@ RUN cargo build --release
 FROM debian:bookworm
 # Install runtime dependencies
 RUN apt-get -y update && \
-    apt-get install -y openssl ca-certificates &&\
+    apt-get install -y openssl ca-certificates tini &&\
```

Comment:
This is not necesasry as we do not use tine currently. Either remove it or consider updating the Dockerfile as per [this guide](https://github.com/krallin/tini?tab=readme-ov-file#using-tini)

Reply:
it would be good to set it as the entrypoint